### PR TITLE
Link to basepath

### PR DESCRIPTION
**Pull request by bldng**
Right now, the sidebar logo always points to / , but it should rather be the basePath if provided. I was not sure what the difference is between path and basePath, so it might need to be adjusted for that.

### CODE REVIEW

File: src/CatalogPropTypes.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ export const pageShape = PropTypes.shape({
   pages: PropTypes.array, // should be arrayOf(page) but that doesn't work
   styles: PropTypes.array.isRequired,
   scripts: PropTypes.array.isRequired,
-  imports: PropTypes.object.isRequired
+  imports: PropTypes.object.isRequired,
+  basePath: PropTypes.string
```
Comment (herrstucki) [inline src/CatalogPropTypes.js]: The pageShape doesn't have a `basePath` prop. It's included in the path.

File: src/components/Menu/Menu.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ class Menu extends React.Component {
     return (
       <div style={currentStyle.bar} >
         <div style={{flex: 1}}>
-          <Link to='/' style={{textDecoration: 'none'}}>
+          <Link to={basePath ? basePath : '/'} style={{textDecoration: 'none'}}>
```
Comment (herrstucki) [inline src/components/Menu/Menu.js]: Maybe it would be better to always set `basePath` during configuration and make it required everywhere else.

Comment (naehrstoff) [inline src/components/Menu/Menu.js]: Yes, definitely better.

File: src/components/Menu/Menu.js [hunk 3]
```diff
@@ -81,6 +81,7 @@ Menu.propTypes = {
   theme: PropTypes.object.isRequired,
   logoSrc: PropTypes.string,
   history: PropTypes.object.isRequired,
+  basePath: PropTypes.string,
```
Comment (naehrstoff) [inline src/components/Menu/Menu.js]: AS per the above, this should then be a required prop

**Pull request merged**